src/game.cpp

```diff
@@ -1429,16 +1419,16 @@ class Game {
 
 	/* GUI stuff
 	 */
-	gui::IGUIStaticText *guitext;          // First line of debug text
+	gui::IGUIStaticText *guitext;          // First line of status/debug text
 	gui::IGUIStaticText *guitext2;         // Second line of debug text
 	gui::IGUIStaticText *guitext3;         // Third line of debug text
-	gui::IGUIStaticText *guitext_info;     // At the middle of the screen
-	gui::IGUIStaticText *guitext_status;
-	gui::IGUIStaticText *guitext_chat;	   // Chat text
+	gui::IGUIStaticText *guitext_info;     // Object info (e.g. "Furnace out of fuel")
+	gui::IGUIStaticText *guitext_notice;   // GUI messages (e.g. "Minimap hidden")
```

Comment:
while it might be a clearer, it adds too many pointless changes to this commit/PR IMO